src/cmdline.rs

```diff
@@ -49,8 +49,8 @@ pub struct InstallConfig {
     pub ignition_hash: Option<IgnitionHash>,
     pub platform: Option<String>,
     pub firstboot_kargs: Option<String>,
-    pub append_kargs: Option<Vec<String>>,
-    pub delete_kargs: Option<Vec<String>>,
+    pub append_kargs: Vec<String>,
+    pub delete_kargs: Vec<String>,
```

Comment:
I was imagining the `unwrap_or_else()` would happen in the immediate caller of `bls_entry_delete_and_append_kargs`, not in the command-line parser, for consistency.  But it's a nit.

Reply:
I did it in the cmdline code so as to avoid the awkward/ambiguous `Option<Vec<String>>` type entirely.